<ui version="4.0" >
 <class>RotationWidget</class>
 <widget class="QWidget" name="RotationWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>336</width>
    <height>144</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" >
   <item>
    <layout class="QVBoxLayout" >
     <item>
      <widget class="QDial" name="x_dial" >
       <property name="toolTip" >
        <string>Rotación sobre X</string>
       </property>
       <property name="statusTip" >
        <string>Rotar la aguja para girar el objeto sobre el eje X</string>
       </property>
       <property name="maximum" >
        <number>359</number>
       </property>
       <property name="pageStep" >
        <number>15</number>
       </property>
       <property name="tracking" >
        <bool>true</bool>
       </property>
       <property name="wrapping" >
        <bool>true</bool>
       </property>
       <property name="notchesVisible" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string>&amp;X</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignCenter</set>
       </property>
       <property name="buddy" >
        <cstring>x_dial</cstring>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" >
     <item>
      <widget class="QDial" name="y_dial" >
       <property name="toolTip" >
        <string>Rotación sobre Y</string>
       </property>
       <property name="statusTip" >
        <string>Rotar la aguja para girar el objeto sobre el eje Y</string>
       </property>
       <property name="maximum" >
        <number>359</number>
       </property>
       <property name="pageStep" >
        <number>15</number>
       </property>
       <property name="tracking" >
        <bool>true</bool>
       </property>
       <property name="wrapping" >
        <bool>true</bool>
       </property>
       <property name="notchesVisible" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2" >
       <property name="text" >
        <string>&amp;Y</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignCenter</set>
       </property>
       <property name="buddy" >
        <cstring>y_dial</cstring>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" >
     <item>
      <widget class="QDial" name="z_dial" >
       <property name="toolTip" >
        <string>Rotación sobre Z</string>
       </property>
       <property name="statusTip" >
        <string>Rotar la aguja para girar el objeto sobre el eje Z</string>
       </property>
       <property name="maximum" >
        <number>359</number>
       </property>
       <property name="pageStep" >
        <number>15</number>
       </property>
       <property name="tracking" >
        <bool>true</bool>
       </property>
       <property name="wrapping" >
        <bool>true</bool>
       </property>
       <property name="notchesVisible" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3" >
       <property name="text" >
        <string>&amp;Z</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignCenter</set>
       </property>
       <property name="buddy" >
        <cstring>z_dial</cstring>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../resource/default.qrc" />
 </resources>
 <connections/>
</ui>
